modules/ossm-installing-multi-cluster-mesh.adoc

```diff
@@ -0,0 +1,179 @@
+$ export ISTIO_VERSION=1.23.2
+----
+
+. Create a namespace called `istio-system` for the first cluster by running the following command at the terminal:
++
+[source,terminal]
+----
+$ oc get ns istio-system --context "${CTX_CLUSTER1}" || kubectl create namespace istio-system --context "${CTX_CLUSTER1}"
+----
+
+. Create a namespace called `istio-system`  for the second cluster:
++
+[source,terminal]
+----
+$ oc get ns istio-system --context "${CTX_CLUSTER2}" || kubectl create namespace istio-system --context "${CTX_CLUSTER2}"
+----
+
+. Create a shared root certificate:
++
+[source,terminal]
+----
+$ openssl genrsa -out root-key.pem 4096
+encrypt_key = no
+prompt = no
+utf8 = yes
+default_md = sha256
+default_bits = 4096
+req_extensions = req_ext
+x509_extensions = req_ext
+distinguished_name = req_dn
+[ req_ext ]
+subjectKeyIdentifier = hash
+basicConstraints = critical, CA:true
+keyUsage = critical, digitalSignature, nonRepudiation, keyEncipherment, keyCertSign
+[ req_dn ]
+O = Istio
+CN = Root CA
+
+openssl req -sha256 -new -key root-key.pem \
+  -config root-ca.conf \
+  -out root-cert.csr
+
+openssl x509 -req -sha256 -days 3650 \
+  -signkey root-key.pem \
+  -extensions req_ext -extfile root-ca.conf \
+  -in root-cert.csr \
+  -out root-cert.pem
+----
+
+. Create an intermediate certificate for the east cluster:
++
+[source,terminal]
+----
+$ openssl genrsa -out east/ca-key.pem 4096
+$east/intermediate.conf
+[ req ]
+encrypt_key = no
+prompt = no
+utf8 = yes
+default_md = sha256
+default_bits = 4096
+req_extensions = req_ext
+x509_extensions = req_ext
+distinguished_name = req_dn
+[ req_ext ]
+subjectKeyIdentifier = hash
+basicConstraints = critical, CA:true, pathlen:0
+keyUsage = critical, digitalSignature, nonRepudiation, keyEncipherment, keyCertSign
+subjectAltName=@san
+[ san ]
+DNS.1 = istiod.istio-system.svc
+[ req_dn ]
+O = Istio
+CN = Intermediate CA
+L = $cluster
+EOF
+
+  openssl req -new -config $east/intermediate.conf \
+    -key $east/ca-key.pem \
+    -out $east/cluster-ca.csr
+
+  openssl x509 -req -sha256 -days 3650 \
+    -CA root-cert.pem \
+    -CAkey root-key.pem -CAcreateserial \
+    -extensions req_ext -extfile $east/intermediate.conf \
+    -in $east/cluster-ca.csr \
+    -out $east/ca-cert.pem
+
+  cat $east/ca-cert.pem root-cert.pem \
+    > $east/cert-chain.pem
+  cp root-cert.pem $east
+----
+
+. Create an intermediate certificate for the west cluster:
++
+[source,terminal]
+----
+openssl genrsa -out $west/ca-key.pem 4096
+$west/intermediate.conf
+[ req ]
+encrypt_key = no
+prompt = no
+utf8 = yes
+default_md = sha256
+default_bits = 4096
+req_extensions = req_ext
+x509_extensions = req_ext
+distinguished_name = req_dn
+[ req_ext ]
+subjectKeyIdentifier = hash
+basicConstraints = critical, CA:true, pathlen:0
+keyUsage = critical, digitalSignature, nonRepudiation, keyEncipherment, keyCertSign
+subjectAltName=@san
+[ san ]
+DNS.1 = istiod.istio-system.svc
+[ req_dn ]
+O = Istio
+CN = Intermediate CA
+L = $cluster
```

Comment:
this needs to go into a separate file called west/intermediate.conf

Reply:
done